src/utils/sanitize-href.js

```diff
@@ -0,0 +1,4 @@
+const sanitizeHref = (href = '', isBeta = false) =>
```

Comment:
why not directly export it?
```suggestion
export default (href = '', isBeta = false) =>
```

Reply:
We could. My head is thinking that the FEC will not generate a title for documentation :smile: but this is not FEC.